### ISSUE TITLE
AltDrag stops working after Remote Desktop session is finish

### ISSUE THREAD

**Issue by pablospe** [stefansundin/altdrag#107]
After remote desktop, altdrag stops working. I need to kill it and start a new instance to make it work again.

This issue is easy to reproduce, just connect to a machine with altdrag running, disconnect remote desktop, and login normally (without remote desktop) to the machine. Altdrag will stop working.

Is there any way to give more details, like a log file that I can share to simplify the debugging?

**Comment by warnerAWESOMEmoore** [comment]
I might be experiencing the same issue. More recently, I've behavior where it locks up and I have to terminate and restart the process. I've been unable to isolate it.

**Comment by benley** [comment]
I'm not sure if this is the exact same issue, but I suspect it is: if the screen resolution changes (independent of a remote session), altdrag immediately hangs.

**Comment by pablospe** [comment]
Yes, it looks like that it is related to the screen resolution changes.

**Comment by dtruebin** [comment]
Meaning, this one is probably a dupe of #34 (there is a number of similar issues open, referencing the oldest one, even though its title is rather misleading).
